bin/import_cv2.py

```diff
@@ -137,8 +131,28 @@ def _maybe_convert_wav(mp3_filename, wav_filename):
 
 
 if __name__ == "__main__":
-    audio_dir = sys.argv[1]
-    tsv_dir = sys.argv[2]
-    print('Expecting your audio from Common Voice to be in: ', audio_dir)
-    print('Looking for *.tsv files (generated by CorporaCreator) in: ', tsv_dir)
-    _preprocess_data(audio_dir, tsv_dir)
+    parser = argparse.ArgumentParser(description='Import CommonVoice v2.0 corpora')
+    parser.add_argument('tsv_dir', help='Directory containing tsv files')
+    parser.add_argument('--audio_dir', help='Directory containing the audio clips - defaults to "<tsv_dir>/clips"')
+    parser.add_argument('--filter_alphabet', help='Exclude samples with characters not in provided alphabet')
+    parser.add_argument('--normalize', action='store_true', help='Converts diacritic characters to their base ones')
+    parser.set_defaults(normalize=False)
+    params = parser.parse_args()
+
+    audio_dir = params.audio_dir if params.audio_dir else os.path.join(params.tsv_dir, 'clips')
+    alphabet = Alphabet(params.filter_alphabet) if params.filter_alphabet else None
+
+    def label_filter(label):
+        if params.normalize:
+            label = unicodedata.normalize("NFKD", label.strip()) \
```

Comment:
So this is just for English for now.

Reply:
This particular option only makes sense for English, yes (as far as I can tell).